src/Dfc.CourseDirectory.WebV2/ServiceCollectionExtensions.cs

```diff
@@ -160,6 +160,7 @@ public static class ServiceCollectionExtensions
             services.Configure<GoogleAnalyticsOptions>(configuration.GetSection("GoogleAnalytics"));
             services.Configure<GoogleTagManagerOptions>(configuration.GetSection("GoogleTagManager"));
             services.AddTransient<SqlDataSync>();
+            services.AddTransient<RouteValuesHelper>();
```

Comment:
Surely this would be scoped, requiring scope for the `HttpContext`?

Reply:
Don't think it will make a difference here but I've changed it